File: Service_eHospital_DongNai_A/Implement/Lst_KeyData_SoThuTuTiepNhanService.cs
namespace Service_eHospital_DongNai_A.Implement
{
    public class Lst_KeyData_SoThuTuTiepNhanService : BaseService<Lst_KeyData_SoThuTuTiepNhan, ILst_KeyData_SoThuTuTiepNhanRepository>
    , ILst_KeyData_SoThuTuTiepNhanService
    {
    private readonly ILst_KeyData_SoThuTuTiepNhanRepository _Lst_KeyData_SoThuTuTiepNhanRepository;
    public Lst_KeyData_SoThuTuTiepNhanService(ILst_KeyData_SoThuTuTiepNhanRepository Lst_KeyData_SoThuTuTiepNhanRepository) : base(Lst_KeyData_SoThuTuTiepNhanRepository)
    {
    _Lst_KeyData_SoThuTuTiepNhanRepository = Lst_KeyData_SoThuTuTiepNhanRepository;
    }
    }
    }

File: CRM/src/app/shared/eHospital_DongNai_A/Lst_KeyData_SoThuTuTiepNhan.service.ts
import { Injectable } from '@angular/core';
import { HttpClient, HttpParams } from '@angular/common/http';
import { environment } from 'src/environments/environment';
import { Lst_KeyData_SoThuTuTiepNhan } from './Lst_KeyData_SoThuTuTiepNhan.model';
import { BaseService } from './Base.service';
@Injectable({
    providedIn: 'root'
})
export class Lst_KeyData_SoThuTuTiepNhanService extends BaseService{
    DisplayColumns001: string[] = ['Save', 'STT', 'KeyType', 'KeyYear', 'KeyMonth', 'KeyDay', 'Segment_1', 'Segment_2', 'Segment_3', 'KeyValue'];

    List: Lst_KeyData_SoThuTuTiepNhan[] | undefined;
    ListFilter: Lst_KeyData_SoThuTuTiepNhan[] | undefined;
    FormData!: Lst_KeyData_SoThuTuTiepNhan;
    constructor(public httpClient: HttpClient) {
        super(httpClient);
        this.Controller = "Lst_KeyData_SoThuTuTiepNhan";
    }
}

